packages/material-ui/src/ClickAwayListener/ClickAwayListener.js

```diff
@@ -114,6 +114,11 @@ ClickAwayListener.propTypes = {
   touchEvent: PropTypes.oneOf(['onTouchStart', 'onTouchEnd', false]),
 };
 
+if (process.env.NODE_ENV !== 'production') {
+  // eslint-disable-next-line
+  ClickAwayListener['propTypes' + ''] = exactProp(ClickAwayListener.propTypes);
```

Comment:
Guess this tricks babel-plugin-remove-prop-types?

Reply:
Yes, you are correct. While I didn't try without it in this specific case.